vendor/engines/saas/app/jobs/analytics/identify_user_job.rb

```diff
@@ -4,7 +4,7 @@ class IdentifyUserJob < AnalyticsJob
 
     def payload(params)
       {
-        user_id: params['current_user']['id'],
+        user_id: params['current_user']['id'] || "Anonymous",
```

Comment:
I'm concerned about the repercussions of identifying anonymous users and how that will affect our intercom bill.

Please do some investigating before we commit to a strategy